apps/aecore/lib/aecore/chain/chain_state.ex

```diff
@@ -79,6 +82,25 @@ defmodule Aecore.Chain.ChainState do
       Enum.all?()
   end
 
+  @spec add_fees_to_block_state(list(), map()) :: map()
+  def add_fees_to_block_state(txs, block_state) do
```

Comment:
What does this function do?

Reply:
it adds the sum of the fees to the given account in the block state